src/renderer/resources/style/main.scss

```diff
@@ -1,5 +1,5 @@
 $fa-font-path: "~font-awesome/fonts/" !default;
-$roboto-font-path: "~materialize-css/fonts/roboto/" !default;
```

Comment:
`materialize-css` moved `fonts` to `dist`.
They also have a `css` in `dist`, not sure if there would be advantages to using that, rather than the uncompiled `scss`?